docs/_includes/common/icon.html

```diff
@@ -180,11 +180,11 @@ <h3>Notice Icons</h3>
         </div>
         <div class="demo__tile">
             <div class="demo__inner">
-                <svg aria-hidden="true" class="icon icon--priority" focusable="false" height="24" width="24">
-                    <use xlink:href="#icon-priority"></use>
+                <svg aria-hidden="true" class="icon icon--attention" focusable="false" height="24" width="24">
+                    <use xlink:href="#icon-attention"></use>
                 </svg>
             </div>
-            <span class="highlight">priority</span>
+            <span class="highlight">attention</span>
```

Comment:
here I wasn't sure for DS4 if we should keep `priority` since it's currently there in the docs now. Not sure how much people are using it though. Either way, I'm fine with changing common/icon to attention 👍

Reply:
Yeah, what we tend to do is update the docs with the new term for new adoptees of ds4, but leave the old classes around for backwards compatibility. We've found that it's best to keep driving everyone forward with new names.

Reply:
Well, we this is a breaking version.

Reply:
Yeah, sometimes even in breaking versions I'll leave some backwards compatibility in where it makes sense. This is one of those cases. I will remove flyout-notice and 'priority' together in v8.

Reply:
What I meant, though, is that we're already telling folks what to use _going forward._ 👍

Reply:
Do you want me to change something?

Reply:
Nope. Let's do it!!